include.js

```diff
@@ -5960,6 +5960,13 @@ let cards = [
     author: 'Jinal Patel',
     githubLink: 'https://github.com/JinalPatel17'
   }
+  {
```

Comment:
Hi @ghazanfarali1994 , you seemed to have omitted a '**_,_**' around here.
As a result of this, the deployment preview won't display anything from the cards object.
![Screenshot (327)](https://user-images.githubusercontent.com/32850163/174992335-92fead8b-194a-4a5f-bfd2-a2e3fca9b95c.png)